app/javascript/controllers/map_controller.js

```diff
@@ -0,0 +1,224 @@
+        shadow.appendChild(ellipse);
+    }
+
+    const background = this.createNS('http://www.w3.org/2000/svg', 'g');
+    background.setAttributeNS(null, 'fill', _color);
+
+    const bgPath = this.createNS('http://www.w3.org/2000/svg', 'path');
+    bgPath.setAttributeNS(null, 'd', 'M27,13.5 C27,19.074644 20.250001,27.000002 14.75,34.500002 C14.016665,35.500004 12.983335,35.500004 12.25,34.500002 C6.7499993,27.000002 0,19.222562 0,13.5 C0,6.0441559 6.0441559,0 13.5,0 C20.955844,0 27,6.0441559 27,13.5 Z');
+
+    background.appendChild(bgPath);
+
+    const border = this.createNS('http://www.w3.org/2000/svg', 'g');
+    border.setAttributeNS(null, 'opacity', '0.25');
+    border.setAttributeNS(null, 'fill', '#000000');
+
+    const borderPath = this.createNS('http://www.w3.org/2000/svg', 'path');
+    borderPath.setAttributeNS(null, 'd', 'M13.5,0 C6.0441559,0 0,6.0441559 0,13.5 C0,19.222562 6.7499993,27 12.25,34.5 C13,35.522727 14.016664,35.500004 14.75,34.5 C20.250001,27 27,19.074644 27,13.5 C27,6.0441559 20.955844,0 13.5,0 Z M13.5,1 C20.415404,1 26,6.584596 26,13.5 C26,15.898657 24.495584,19.181431 22.220703,22.738281 C19.945823,26.295132 16.705119,30.142167 13.943359,33.908203 C13.743445,34.180814 13.612715,34.322738 13.5,34.441406 C13.387285,34.322738 13.256555,34.180814 13.056641,33.908203 C10.284481,30.127985 7.4148684,26.314159 5.015625,22.773438 C2.6163816,19.232715 1,15.953538 1,13.5 C1,6.584596 6.584596,1 13.5,1 Z');
+
+    border.appendChild(borderPath);
+
+    const maki = this.createNS('http://www.w3.org/2000/svg', 'g');
+    maki.setAttributeNS(null, 'transform', 'translate(6.0, 7.0)');
+    maki.setAttributeNS(null, 'fill', '#FFFFFF');
+
+    const circleContainer = this.createNS('http://www.w3.org/2000/svg', 'g');
+    circleContainer.setAttributeNS(null, 'transform', 'translate(8.0, 8.0)');
+
+    const circle1 = this.createNS('http://www.w3.org/2000/svg', 'circle');
+    circle1.setAttributeNS(null, 'fill', '#000000');
+    circle1.setAttributeNS(null, 'opacity', '0.25');
+    circle1.setAttributeNS(null, 'cx', '5.5');
+    circle1.setAttributeNS(null, 'cy', '5.5');
+    circle1.setAttributeNS(null, 'r', '5.4999962');
+
+    const circle2 = this.createNS('http://www.w3.org/2000/svg', 'circle');
+    circle2.setAttributeNS(null, 'fill', '#FFFFFF');
+    circle2.setAttributeNS(null, 'cx', '5.5');
+    circle2.setAttributeNS(null, 'cy', '5.5');
+    circle2.setAttributeNS(null, 'r', '5.4999962');
+
+    circleContainer.appendChild(circle1);
+    circleContainer.appendChild(circle2);
+
+    page1.appendChild(shadow);
+    page1.appendChild(background);
+    page1.appendChild(border);
+    page1.appendChild(maki);
+    page1.appendChild(circleContainer);
+
+    svg.appendChild(page1);
+
+    svg.setAttributeNS(null, 'height', `${defaultHeight * _scale}px`);
+    svg.setAttributeNS(null, 'width', `${defaultWidth * _scale}px`);
+
+    _element.appendChild(svg);
+
+    return _element
+  }
```

Comment:
Cool. Any strong reason to rewrite this here rather than importing it from mapbox.js?

Reply:
I don't remember why I didn't import it haha (I remember thinking about it though). This is only meant to be temporary anyways, so I think it was faster just copy and pasting.

Reply:
oki :)